### DISCUSSION TITLE
How do I do "inline mode' CSS in JS w/ Next?

### DISCUSSION THREAD

**Discussion by devuxer** [ant-design/ant-design#43266] [Q&A]
For a NextJS app, where do I put the code that is shown under [customize theme > SSR](https://ant.design/docs/react/customize-theme#server-side-render-ssr)  > Inline mode?

**Comment by zombieJ** [comment]
Your link already give it:
https://ant.design/docs/react/customize-theme#inline-mode

<img width="458" alt="截屏2023-06-29 15 05 24" src="https://github.com/ant-design/ant-design/assets/5378891/2ef59cd0-81d5-4c78-9893-f830d66885c0">

**Comment by devuxer** [comment]
@zombieJ , 

I see the code, but I don't know what file to put it in within my NextJS project.

Does it go in _app.tsx? _document.tsx? Somewhere else?

Do I need to use `dangerouslySetInnerHTML`?
